locales/en/public.json

```diff
@@ -12,6 +12,23 @@
     "OPEN_SOURCE_LICENSE": "Open Source License",
     "VERSION": "version"
   },
+  "CHART_CARD": {
+    "BUTTONS": {
+      "CREATE": {
+        "LABEL": "Create"
```

Comment:
I notice these aria-labels have a different scheme from some other aria labels in the file
e.g.
```json
  "DateTimePicker": {
    "ARIA_LABELS": {
      "DISPLAY_SELECTED_DATETIME": "Displayed selected datetime",
      "TABS": "Select a date or time tab"
    },
    "SELECTED_DATETIME": "Selected DateTime"
  },
  "MeridiemPicker": {
    "ARIA_LABELS": {
      "LISTBOX": "Select AM or PM"
    }
  },
```
Maybe either change this or we should change the other ones to keep consistency?

Reply:
We probably need to document key format somewhere too

Reply:
You mean just change the way that the key is split up? Or change the actual localized text?

Reply:
I mean the way the key is split up. Maybe we should have 
```json
"DateTimePicker": {
    "TABS": {  
          "LABEL": "Select a date or time tab"
     }
```
in the same format in this PR that you have done here. If so, then that's okay to change in a separate refactor PR.

Just keeping a consistent key format.

Reply:
Sure, I don't really mind either way. I think there probably are/will be instances where the `aria-label` matches some text that is visually rendered, so specifying `ARIA_LABELS` as part of the key maybe doesn't make the most sense.